hamilflow/models/pendulum.py

```diff
@@ -0,0 +1,146 @@
+import math
+from functools import cached_property
+from typing import Any, Dict, Optional, Union
+
+import numpy as np
+import pandas as pd
+from numpy.typing import ArrayLike
+from pydantic import BaseModel, Field, computed_field
+from scipy.special import ellipj, ellipk
+
+
+class PendulumSystem(BaseModel):
+    r"""The params for the pendulum
+
+    :param float omega0: $\omega_0 \coloneqq \sqrt{\frac{U}{I}} > 0$, frequency
```

Comment:
type in docstrings is not required. The type of the var can be read directly through the type hints. The generated website also uses the type hint as types.

<img width="718" alt="Screenshot 2024-03-21 at 12 23 28" src="https://github.com/kausalflow/hamilflow/assets/663798/c681ecd1-bbce-4429-a090-00b5f6229f87">

Since linting docstring is hard, including types in docstring takes a lot of effort to manage. It provides little benefits but maybe inconsistent with the type hint during refactor practices.

```suggestion
    :param omega0: $\omega_0 \coloneqq \sqrt{\frac{U}{I}} > 0$, frequency
```

Reply:
The types were missing on my local `mkdocs` when I did not specify them. Now I see they are available on the deployed `mkdocs`.

Reply:
Interesting. It works on my local build. Maybe the installed packages are a bit outdated? Maybe run `poetry install` to install from lock and try again.